chef/data_bags/crowbar/migrate/network/021_add_mtu_os_sdn.rb

```diff
@@ -0,0 +1,15 @@
+def upgrade(ta, td, a, d)
+  unless a["networks"]["os_sdn"].key? "mtu"
+    a["networks"]["os_sdn"]["mtu"] = ta["networks"]["os_sdn"]["mtu"]
+  end
+
+  return a, d
+end
+
+def downgrade(ta, td, a, d)
+  unless a["networks"]["os_sdn"].key? "mtu"
```

Comment:
Sorry, I replied to fast. I was thinking of the upgrade, not the downgrade. I see your point.